### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by mikaEz**

### CODE REVIEW

File: .github/workflows/test..yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (schurzi) [inline .github/workflows/test..yml]: duplicate dot ("..") in filename ;)

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -3,10 +3,7 @@ name: linting
 
 on: [push, pull_request]
 
-env:
-  TEMPLATE_REPOSITORY: T-Systems-MMS/terraform-template
-
 jobs:
   call-linting:
-    if: github.repository != '$TEMPLATE_REPOSITORY'
+    if: github.repository != 'T-Systems-MMS/terraform-template'
```
Comment (schurzi) [inline .github/workflows/linting.yml]: maybe leave this check out and just disable actions for this repository? Or do we need to run some actions here?

File: .github/workflows/test..yml (hunk 1 above)
Comment (mikaEz) [reply .github/workflows/test..yml]: fixed

File: .github/workflows/linting.yml (hunk 2 above)
Comment (mikaEz) [reply .github/workflows/linting.yml]: Change Actions policy to "Disable actions" setting for this repository.

**Review by schurzi — APPROVED** [review]

**Pull request merged**